js/select.js

```diff
@@ -265,13 +265,21 @@
       else this._selectValue(value);
     }
     _getSelectedOptions() {
-      return Array.prototype.map.call(this.el.selectedOptions, (realOption) => realOption);
+      return Array.prototype.filter.call(this.el.selectedOptions, (realOption) => realOption);
```

Comment:
eh wait, how does this work? 😅

Reply:
https://stackoverflow.com/questions/11821261/how-to-get-all-selected-values-from-select-multiple-multiple

I first had troubles with setting things in the DOM. If you refresh your browser, the selected Values are kept in the cache and are not set in the DOM. So I fond the solution here. And since we drop support for IE it works good enough. If it is **filter** or **map** does not really matter it only converts the DataType to an Array.

Reply:
https://caniuse.com/?search=selectedOptions

Reply:
I think this could also be changed to `Array.from()` if you want.
https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/from
Have you tested it? When should we merge?

Reply:
> If it is **filter** or **map** does not really matter it only converts the DataType to an Array.

I see. That's funny that selectedOptions is not an Array haha 😆

Reply:
Yes it is some DOM specific array which has to be converted for some odd reason...the same issue is with querySelectorAll Method.